src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -34,6 +34,11 @@
             <argument type="service" id="validator" />
         </service>
 
+        <service id="messenger.middleware.handles_recorded_messages" class="Symfony\Component\Messenger\Middleware\HandlesRecordedMessagesMiddleware" abstract="true">
```

Comment:
Looking at the names of other middlewares, this one maybe should be called `messenger.middleware.record_messages`